demos/model-deployment-with-streaming/1-event-generator.ipynb

```diff
@@ -265,6 +265,14 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
+      "2020-08-09 10:44:03,931 [info] Disconnected while attempting to send. Recreating connection: {'e': <class 'BrokenPipeError'>}\n",
+      "2020-08-09 10:44:03,950 [info] Disconnected while attempting to send. Recreating connection: {'e': <class 'BrokenPipeError'>}\n",
+      "2020-08-09 10:44:03,968 [info] Disconnected while attempting to send. Recreating connection: {'e': <class 'BrokenPipeError'>}\n",
+      "2020-08-09 10:44:03,984 [info] Disconnected while attempting to send. Recreating connection: {'e': <class 'BrokenPipeError'>}\n",
+      "2020-08-09 10:44:03,999 [info] Disconnected while attempting to send. Recreating connection: {'e': <class 'BrokenPipeError'>}\n",
+      "2020-08-09 10:44:04,014 [info] Disconnected while attempting to send. Recreating connection: {'e': <class 'BrokenPipeError'>}\n",
+      "2020-08-09 10:44:04,029 [info] Disconnected while attempting to send. Recreating connection: {'e': <class 'BrokenPipeError'>}\n",
+      "2020-08-09 10:44:04,045 [info] Disconnected while attempting to send. Recreating connection: {'e': <class 'BrokenPipeError'>}\n",
```

Comment:
@gilad-shaham did you intentionally/knowingly insert these output error logs as part of the PR changes? Perhaps you should remove them?

Reply:
It just depends on whether the notebook runs for the first time or reruns. We can fix it.